app/src/main/res/layout/event_card.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Are you looking for feed_card.xml?

Reply:
Yeah.. I found that after opening the Pull Request..